physics/dynamic_frame.hpp

```diff
@@ -24,7 +37,51 @@ class DynamicFrame {
   virtual Vector<Acceleration, ThisFrame> GeometricAcceleration(
       Instant const& t,
       DegreesOfFreedom<ThisFrame> const& degrees_of_freedom) const = 0;
+
+  // The definition of the Frenet frame of a free fall trajectory in |ThisFrame|
+  // with the given |DegreesOfFreedom| at the given |Instant|.  Unless
+  // specialized otherwise by child classes, the member functions of the result
+  // fail when called at instants other than |t|.
+  virtual Rotation<Frenet<ThisFrame>, ThisFrame> FrenetFrame(
+      Instant const& t,
+      DegreesOfFreedom<ThisFrame> const& degrees_of_freedom) const;
+};
+
+// An inertial frame.
+template<typename OtherFrame, typename ThisFrame>
+class InertialFrame : public DynamicFrame<OtherFrame, ThisFrame> {
+ public:
+  InertialFrame(Velocity<OtherFrame> const& velocity,
+                Position<OtherFrame> const& origin_at_epoch,
+                Instant const& epoch,
+                OrthogonalMap<OtherFrame, ThisFrame> const& orthogonal_map,
+                std::function<Vector<Acceleration, OtherFrame>(
+                    Instant const& t,
+                    Position<OtherFrame> const& q)> gravity);
```

Comment:
Maybe, but then we'd really need dummy ephemerides so the tests remain palatable, and that's a job for another CL. `InertialFrame` exists for testing purposes only anyway.